data/presets/amenity/health_post.json

```diff
@@ -0,0 +1,42 @@
+{
+    "icon": "fas-house-medical-flag",
+    "fields": [
+        "name",
+        "operator",
+        "address",
+        "building_area_yes",
+        "opening_hours"
+    ],
+    "moreFields": [
+        "{@templates/poi}",
+        "healthcare/health_post",
```

Comment:
![image](https://github.com/openstreetmap/id-tagging-schema/assets/5821688/874a8dc9-ff46-4d49-a0b6-06c3ffd3a655)

Kinda works, but not exactly what I wanted. Other healthcare values still showing up (in black), while those listed in health_post show up as (blue).